CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change history for stripes-cli
 
+## [1.12.0](https://github.com/folio-org/stripes-cli/tree/v1.12.0) (2019-05-15)
+
+* Upgrade stripes-testing dependency. Refs STCLI-137.
```

Comment:
Didn't you already do this in v1.11.0?

Reply:
I think I got my cart before my horse. In fact, there _is_ another stripes-testing release and it _should_ be included here as a dep in package.json, but it isn't. I'll fix that. Good catch. Thank you.